back-end/handlers/hostHandler.ts

```diff
@@ -13,13 +13,11 @@ import Player from '../models/Player.ts';
 import IPlayer from '../interfaces/IPlayer.ts';
 import PreGameSettings from '../models/PreGameSettings.ts';
 
-var GameId ;
```

Comment:
This global variable was causing issues when simultaneous games were happening. Basically, we can't set a `GameId` variable like this, because we don't know which game might be sending requests. This game ID will also only exist for the lifetime of the server, and it's stored in local memory - we want to make sure everything that should persist is stored in the database.